views/default/icon/user/default.php

```
@@ -60,22 +60,12 @@
 	$use_hover = $vars['hover'];
 }
 
-$spacer_url = elgg_get_site_url() . '_graphics/spacer.gif';
-
-$icon_url = elgg_format_url($user->getIconURL($size));
-$params = array(
-	'src' => $spacer_url,
+$icon = elgg_view('output/img', array(
+	'src' => $user->getIconURL($size),
 	'alt' => $name,
 	'title' => $name,
 	'class' => $img_class,
-	'style' => "background: url($icon_url) no-repeat;",
-);
-if (in_array($size, array('small', 'tiny', 'topbar'))) {
-	$icon_sizes = elgg_get_config('icon_sizes');
-	$params['width'] = $icon_sizes[$size]['w'];
-	$params['height'] = $icon_sizes[$size]['h'];
-}
-$icon = elgg_view('output/img', $params);
+));
 
 $show_menu = $use_hover && (elgg_is_admin_logged_in() || !$user->isBanned());
 
```

The only difference with this file and the default one is that this one hides the fact that user is banned. Why do we want to hide it?